src/services/docs.ts

```diff
@@ -9,13 +10,14 @@ export async function init(): Promise<void> {
     try {
         jsonDocs = JSON.parse(docs);
     } catch (err) {
-        console.error("[ITEM-STORE] Invalid docs", err);
+        logger.error("Invalid docs");
+        logger.error(err);
```

Comment:
interesting. @tkjwa how should we handle the case where we want to log an error object, potentially with the stack trace, is this right?

Reply:
IMO we don't need a full stack trace if the error is handled properly, which mean it is a known code path. Then in case of unhandled error i guess the js engine will log the stack trace in the output